app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,12 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
```

Comment:
There is an error when the user tries to see the posts index page. Kindly see the screenshot below.
![posts](https://user-images.githubusercontent.com/35504149/174405273-622ef924-df2b-4520-bba7-3336e0579b09.png)